### PR TITLE
splitRef-0.0.2.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/2775fd4506cd499addb8 for a full test report.

File: easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version = '0.0.2'
+
+homepage = 'http://www.sph.umich.edu/csg/yli/splitRef/index.html'
+description = """ splitRef splits a reference haplotype file into smaller files 
+ with subsets of markers. The current version is a pre-release. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.sph.umich.edu/csg/yli/splitRef/download/']
+sources = ['%s.V%s.tgz' % (name, version.replace(".",""))]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb]: space after comma, remove final line; @boegel: ready for test/merge?

File: easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    'dirs': ["examples"]
+}
+
+# add folder splitRef_V002 to PATH
+modextrapaths = { 
+    'PATH': '%s_V%s' % (name, version.replace(".",""))
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb]: remove last line, it has the nice benefit of triggering a recent jenkins test (if/when that works!)

lgtm , put this in REVIEW plz!

File: easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+description = """ splitRef splits a reference haplotype file into smaller files 
+ with subsets of markers. The current version is a pre-release. """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.sph.umich.edu/csg/yli/splitRef/download/']
+sources = ['%s.V%s.tgz' % (name, version.replace(".", ""))]
+
+sanity_check_paths = {
+    'files': ["splitRef_V002/splitRef.pl"],
```
Comment (boegel) [inline easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb]: @pescobar hardcoded version, please use:

```python
version = '0.0.2'
altver = version.replace(',', '')
...
    'files': ['splitRef_%s/splitRef.pl' % altver],
```

File: easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    'dirs': ["examples"]
+}
+
+# add folder splitRef_V002 to PATH
+modextrapaths = { 
+    'PATH': '%s_V%s' % (name, version.replace(".", ""))
```
Comment (boegel) [inline easybuild/easyconfigs/s/splitRef/splitRef-0.0.2.eb]: use `altver`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/29dfc1f055f676e15f8515342c3a85ca for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/6b6c7b9f06880efc90cf64fb75b73c13 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/c02f4a32715b276b0cb78c90f891a8af for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login18.cluster.bc2.ch - Linux centos 6.5, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.6.6
See https://gist.github.com/ccb2b7bea2494cb0bec9f1ab0737de2c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2160.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/29850a6e76d110a8f07a1534cb9e0747 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**